acceptance-tests/Makefile.am

```diff
@@ -2425,4 +2445,13 @@ $(CORECLR_PATH)%.exe: $(CORECLR_PATH)%.cs
 test-runner.exe: $(top_srcdir)/mono/tests/test-runner.cs
 	$(MCS) -debug -out:$@ $<
 
-CLEANFILES = $(CORECLR_TESTSI_CS) $(CORECLR_TESTSI_IL) *.dll *.exe *.mdb
+Tests:
+	mkdir Tests
```

Comment:
Maybe GCStressTests to make it clear what's in there?
I've also been building the other .exe's right next to their .cs in the coreclr repo, any reason why we don't want to do this here?